dvc/ignore_templates/java

```diff
@@ -0,0 +1,23 @@
+# Compiled class file
```

Comment:
Hm, I also wonder if these are really worth stealing directly from https://github.com/github/gitignore . They would be hard to maintain, really. But I'm not able to see a better way right now...